DBM-GUI/modules/MainFramePrototype.lua

```diff
@@ -251,6 +254,7 @@ function frame:CreateTab(tab)
 	self.tabs[i] = tab
 	DBM_GUI:CreateNewFauxScrollFrameList()
 	---@class DBMOptionsFrameTabButtonTemplate: Button
+	---@field Text FontString -- (no clue where this field is even from?)
```

Comment:
The field is gained from the template mixin.

Reply:
Ah, never seen that one before, I'll fix that upstream